### PR TITLE
Merge ARROWHEAD-6.3.0.0 into master

### PR DESCRIPTION
**Pull request by marcos-iov**
*rit:new-configs*

### CODE REVIEW

**Comment by marcos-iov** [comment]
pipeline:run

File: src/test/java/co/rsk/federate/BtcToRskClientTest.java [hunk 1]
```diff
@@ -2432,6 +2470,76 @@ void updateBridge_when_does_not_hasBetterBlockToSync_updates_headers_coinbase_tr
...
+        btcToRskClient.updateBridge();
+
+        verify(btcToRskClient, never()).updateBridgeBtcBlockchain();
+        verify(btcToRskClient, never()).updateBridgeBtcCoinbaseTransactions();
+        verify(btcToRskClient, never()).updateBridgeBtcTransactions();
+        verify(federatorSupport, never()).sendUpdateCollections();
+    }
+
+    @Test
+    void updateBridge_noUpdateBridgeConfigDefined_shouldCallUpdatesBridgeMethods() throws Exception {
```
Comment (jeremy-then) [inline src/test/java/co/rsk/federate/BtcToRskClientTest.java:2508]: nit:

```suggestion
    void updateBridge_noUpdateBridgeConfigDefined_shouldOnlyCallSendUpdateCollecdtions() throws Exception {
```

**Review by jeremy-then — APPROVED** [review]
LGTM.

**Review by julia-zack — DISMISSED** [review]

**Review by julia-zack — APPROVED** [review]

**Review by jeremy-then — APPROVED** [review]
LGTM.

**Review by nathanieliov — APPROVED** [review]

**Pull request merged**